front/app/components/FiltersModal/TopBar.tsx

```diff
@@ -91,16 +92,16 @@ interface Props {
   className?: string;
 }
 
-const TopBar = memo<Props>(({ onClose, onReset, className }) => {
+const TopBar = ({ onClose, onReset, className }: Props) => {
   return (
     <Container className={className}>
       <TopBarInner>
         <Left>
           <CloseButton onClick={onClose}>
-            <CloseIcon
-              title={<FormattedMessage {...messages.a11y_closeFilterPanel} />}
-              name="close"
-            />
+            <CloseIcon name="close" />
```

Comment:
Or, at least, maybe we can have the `CloseButton` component abstracted in a way that incorporates accessibility? Then when it's being reused we can make sure we always have an appropriate message?